### ISSUE TITLE
Contrast issue on table headers under dark mode

### ISSUE THREAD

**Issue by deniak** [w3c/aria#2397]
The table headers are failing the contrast ratio requirement under the dark mode.

This is caused by the following rule in [common.css](https://github.com/w3c/aria/blob/main/common/css/common.css#L92):
```css
th{
	background-color:#eee;
}
```

So that rule should either be removed or updated to support the dark mode.

Initially reported [here](https://www.linkedin.com/posts/ramy-gamal-797939157_w3c-web-accessibility-initiative-wai-please-activity-7269579512852463616-Xkoj/).

**Comment by pkra** [comment]
Duplicate of #2394 which itself was a duplicate of #2226
